### ISSUE TITLE
Training without gpu does not use learning rate decrease (lr_reducer)

### ISSUE THREAD

**Issue by Hrovatin** [theislab/scarches#30]
If I see correctly training without gpu does not use lr_reducer parameter/does not decrease learning rate when proceeding with training: 
https://github.com/theislab/scarches/blob/6d5f4ad583e981fb986090c43cee129c86f5c7d2/scarches/models/cvae.py#L846
https://github.com/theislab/scarches/blob/6d5f4ad583e981fb986090c43cee129c86f5c7d2/scarches/models/scarches.py#L411

Also, some other packages include saving and plotting of loss over training so that user can quickly evaluate if training went well. (scVI saves loss and provides plotting function, CellBender remove background saves the loss plot image, which is even more convenient).
